custom/conf/app.example.ini

```diff
@@ -175,6 +175,32 @@ RUN_MODE = ; prod
 ;OFFLINE_MODE = false
 ;DISABLE_ROUTER_LOG = false
 ;;
+;; TLS Settings: Either ACME or manual
+;; (Other common TLS configuration are found before)
+;ENABLE_ACME = false
+;;
+;; ACME automatic TLS settings:
+;;
+;; ACME directory URL (e.g. LetsEncrypt's staging/testing URL: https://acme-staging-v02.api.letsencrypt.org/directory)
+;; Leave empty to default to LetsEncrypt's (production) URL
+;ACME_URL =
+;;
+;; If using LetsEncrypt please read their TOS and manually change this setting to true
+;LETSENCRYPT_ACCEPTTOS = false
```

Comment:
I guess we can rename this to `ACME_ACCEPTTOS`. Other [public CAs](https://github.com/acmesh-official/acme.sh/wiki/Server) will also have TOSs.

Reply:
Thought I guess an argument could be made to just auto-accept the TOS and document "by using this feature, you accept the TOS of the ACME provider". I'm certain some ACME implementations like `acme.sh` do it like that.

Reply:
Reviewing the certmagic code, I think you are right. As long as there is a TOS available for the CA it will prompt and ask the user to accept it. I will try and get a phony TOS on my step-ca and confirm how it works.

If it's like that, we can be less aggressive with the TOS check since it will give a prompt if needed. It can be a semi-hidden option so that we default to certmagic's logic.

Reply:
After some experimentation, I think it's better to add a TOS check in [certmagic's PreCheck](https://github.com/caddyserver/certmagic/blob/b6b3db32bcd5f87d56bc0fba1db708499259653a/acmemanager.go#L220-L230). There are no exposed functions to easily get the Acme Directory and TOS from it, and I don't feel like implementing that part manually.

Reply:
In any case, I suggest renaming the option to `ACME_ACCEPTTOS` so it can be used with the zerossl provider too.